src/main/java/com/eharmony/pho/hbase/PhoenixHBaseDataStoreApiImpl.java

```diff
@@ -140,4 +142,20 @@ private void closeConnectionSafe(Connection conn) {
         }
     }
 
+	@Override
+	public <T> T save(T entity, List<String> selectedFields) {
+		Connection conn = null;
+		try {
+			conn = PhoenixConnectionManager.getConnection(connectionUrl);
+			QueryUpdateBuilder updateBuilder = QueryUpdateBuilder.builderFor(entity).update(selectedFields);
+			T returnEntity = (T) queryExecutor.save(updateBuilder.build(), conn);
+			conn.commit();
+			return returnEntity;
+		} catch (Exception ex) {
+			throw new RuntimeException(ex);
```

Comment:
I would let the consumer of this api log the message. Exception handling is the responsibility of the consumer.